klarna-on-demand/src/test/java/com/klarna/ondemand/ContextTest.java

```diff
@@ -1,21 +1,28 @@
 package com.klarna.ondemand;
 
-import org.junit.Assert;
+import org.junit.After;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 public class ContextTest {
 
     @Rule
     public ExpectedException thrown = ExpectedException.none();
 
+    @After
```

Comment:
I know, but it's not good to do the cleanup in the test, incase there is Exception inside the test, the cleanup will not happened